website/content/docs/auth/saml.mdx

```diff
@@ -0,0 +1,158 @@
+       bound_subjects="*@hashicorp.com" \
+       bound_subjects_type="glob" \
+       token_policies="writer" \
+       bound_attributes=group="admin" \
+       ttl=1h
+   ```
+
+   This role authorizes users that have a subject with an `@hashicorp.com` suffix and
+   are in the `admin` group to authenticate. It also gives the resulting Vault token a
+   time-to-live of 1 hour and the `writer` policy.
+
+   For the complete list of configuration options, please see the
+   [API documentation](/vault/api-docs/auth/saml).
+
+### Assertion Consumer Service URLs
+
+The [`acs_urls`](/vault/api-docs/auth/saml#acs_urls) configuration parameter determines
+where the SAML Response will be sent after users authenticate with the configured SAML
+Identity Provider in their browser.
+
+The values provided to Vault must:
+
+- Match or be a subset of the configured values for the SAML application within the
+  configured Identity Provider.
+- Be directed to the auth method's [Assertion Consumer Service
+  Callback](/vault/api-docs/auth/saml#assertion-consumer-service-callback) API.
+
+<Note>
+  It is highly recommended and enforced by some identity providers to TLS-protect the
+  Assertion Consumer Service URLs. A warning will be returned from Vault if any of the
+  configured Assertion Consumer Service URLs are not protected by TLS.
+</Note>
+
+To support a single auth method mount being used across Vault [replication](/vault/docs/enterprise/replication)
+clusters, `acs_urls` supports configuration of multiple values. For example, to support
+SAML authentication on a primary and secondary Vault cluster, the following `acs_urls`
+configuration could be given:
+
+   ```shell-session
+   $ vault write auth/saml/config \
+      acs_urls="https://primary.vault/v1/auth/saml/callback,https://secondary.vault/v1/auth/saml/callback"
+   ```
+
+The Vault UI and CLI will automatically request the proper Assertion Consumer Service URL
+for the cluster they're configured to communicate with. This means that the entirety of the
+authentication flow will stay within the local cluster.
+
+### Bound Attributes
+
+Once the user has been authenticated the authorization flow will validate
+that both the [`bound_subject`](/vault/api-docs/auth/saml#bound_subject) and
+[`bound_attributes`](/vault/api-docs/auth/saml#bound_attributes) match expected
+values configured for the role. This can be used to restrict access to Vault for
+a subset of users in the SAML Identity Provider.
+
+For example, a role with `bound_subject=*@hashicorp.com` and
+`bound_attributes=groups=support,engineering` will only authorize users whose subject has
+an `@hashicorp.com` suffix and that are in either the `support` or `engineering` group.
+
+Whether it should be an exact match or interpret `*` as a wildcard can be
+controlled by the [`bound_subject_type`](/vault/api-docs/auth/saml#bound_subject_type) and
+[`bound_attributes_type`](/vault/api-docs/auth/saml#bound_subject_type) parameters.
```

Comment:
```suggestion
```

Suggest removing this as we've already referred folks to the API docs for more information and we aren't really providing additional information by repeating the note about wildcard searches.